anasymod/structures/module_clk_manager.py

```diff
@@ -56,14 +56,15 @@ def __init__(self, scfg: StructureConfig):
 
 `ifdef SIMULATION_MODE_MSDSL
 	// emulator clock sequence
-	logic emu_clk_2x_state = 1'b0;

```

Comment:
Setting the initial value of signal like this doesn't simulate properly in Icarus Verilog when the signal is updated in a sequential block.  As a result, I updated the code to initialize the signals in ``initial`` blocks.